x/bank/types/send_authorization.go

```diff
@@ -58,6 +53,10 @@ func (a SendAuthorization) Accept(ctx sdk.Context, msg sdk.Msg) (authz.AcceptRes
 		return authz.AcceptResponse{}, sdkerrors.ErrUnauthorized.Wrapf("cannot send to %s address", toAddr)
 	}
 
+	if limitLeft.IsZero() {
+		return authz.AcceptResponse{Accept: true, Delete: true}, nil
+	}
```

Comment:
Is this necessary? Can't we just let it return the full response where limitLeft is zero?

Reply:
It cannot be deleted if it is updated. So I don't think so.

Reply:
Ahh gotcha, didn't see the difference there